coopTilleuls/CoopTilleulsMigrationBundle
# [RFR] Fix compatibility for Symfony 3.3 about autowire

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #1
| License       | MIT

- Remove useless `alias` option on loader tags
- Remove useless `priority` option on loader tags
- Update documentation
- Update tests
- Auto-configure loaders/transformer using `ContainerBuilder::registerForAutoconfiguration`
- Use [service locator](http://symfony.com/blog/new-in-symfony-3-3-service-locators) on loaders & transformers
- Connect entity & its transformer through an annotation

**BC break: this bundle (in beta) is no longer available for Symfony 2.3, 2.8 or 3.0. It's only available for 3.3+ because of `symfony/dependency-injection:~3.3` dependency. Any suggestion to fix this BC break would be appreciated.**
